spec.bs

```diff
@@ -774,8 +798,11 @@ directly; for authentication the extension can only be accessed via
       :  <dfn>topOrigin</dfn> member
       :: The origin of the top-level frame. Only valid at authentication time.
 
+      :  <dfn>payeeName</dfn> member
+      :: The payee name, if present, that was displayed to the user. Only valid at authentication time.
```

Comment:
What does "Only valid at authentication time." mean?

Reply:
This parameter, like the others in [AuthenticationExtensionsClientInputs](https://w3c.github.io/webauthn/#dictdef-authenticationextensionsclientinputs), is only passed to webauthn via this extension during a credential authentication (i.e. at payment), and not for a credential creation (registration). There's a TODO above to clean up this quirk of this payment webauthn extension.

Reply:
I think I am reacting to the word "valid." What if you say "Only used at authentication time; not registration." Or "Only passed to the authenticator at authentication time; not registration."

Reply:
Done, I like the former!